### PR TITLE
modify submissions, add HelloWorld

### PR DESCRIPTION
**Pull request by martinbagic**

### CODE REVIEW

**Comment by s-will** [comment]
Did you find the minor problem with HelloWorld (have a look at the Travis CI report   continuous-integration/travis-ci/pr — The Travis CI build failed   --->Details
It shows (among many other things) the difference between the expected output and the output of your script.

**Comment by martinbagic** [comment]
Sorry, I'm still finding my way around GitHub. If I got it right, the lowercase w in _world_ was the problem? And now the build is failing because it expects the solution to A2 as well?

**Comment by s-will** [comment]
Hi @martinbagic, you are right. The tests are now passing for submissions A0 and A1. I already activated the A2 test, so this one is now failing - don't worry ;)
Are A0 and A1 ready for review? Then, please invite me. (Request a review)

File: A0/martinbagic-HelloWorld.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+# A0, Hello World
+
+import sys
```
Comment (s-will) [inline A0/martinbagic-HelloWorld.py]: rather use argparse (even here, it is pretty easy to use)

File: A1/martinbagic-WordCount.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# A1, Word Count
+
+import sys
```
Comment (s-will) [inline A1/martinbagic-WordCount.py]: again: use argparse for the command line parsing; from here it really pays off

File: A1/martinbagic-WordCount.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        sorted_count = get_sorted_count(words,unique_words)
+        for word,count in sorted_count:
+            print (word,str(count),sep="\t")
+    else: # prints unique and total word counts
+        unique_word_count = len(unique_words)
+        total_word_count = len(words)
+        print (unique_word_count,"/",total_word_count)
+
+
+filename, settings = get_cmd_args(sys.argv)
```
Comment (s-will) [inline A1/martinbagic-WordCount.py]: protect this from execution, if your file is imported as module:
Use:
```
if __name__=="__main__":
```

Moreover, introduce a main function

**Comment by martinbagic** [comment]
Made the changes. Do you find the comments to be a bit of an overkill?

**Pull request merged**